src/util/index_tree.ts

```diff
@@ -526,12 +568,12 @@ export function traverse<T extends IndexTreeNode<T>>(
 /**
  * `traverseAll` traverses the whole tree (include tombstones) with postorder traversal.
  */
-function traverseAll<T extends IndexTreeNode<T>>(
+export function traverseAll<T extends IndexTreeNode<T>>(
   node: T,
   callback: (node: T, depth: number) => void,
   depth = 0,
 ) {
-  for (const child of node._children) {
+  for (const child of node.allChildren) {
```

Comment:
It seems `allChildren()` does shallow copy of `_children` and I wonder whether it is necessary to use the shallow copy.
Is it intended to make the meaning of `_children` more clear?